simbad/version.py

```diff
@@ -2,7 +2,7 @@
 
 # Maintain sematantic versioning. Further information can
 # be found here [http://semver.org/]
-__version_info__ = (0, 1, 9)
+__version_info__ = (0, 1, 11)
```

Comment:
Did we forget an update or why jump by 2?

Reply:
We forgot to update the version number to 0, 1, 10. Seeing as i'm going to make a release for this pretty soon I thought I might as well move it to 0, 1, 11